.travis.yml

```diff
@@ -1,5 +1,9 @@
 language: python
-python: 2.7
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+  - "nightly"
```

Comment:
Don't understand the point of this change. Which Python versions are being tested is controlled by the tox configuration. This part of the Travis configuration would only control what Python version tox itself was run as. If Travis is installing nightly 3.6 builds, then need to add the label for whatever that is in tox configuration.